### ISSUE TITLE
Improve README.md Documentation

### ISSUE THREAD

**Issue by Kilgamesh** [Kilgamesh/ultima#56]
- Update links @Kilgamesh 
- Fix spacing between lines
- Add link to CONTRIBUTING.md file
- Check for any spelling, grammatical or pronunciation errors

**Comment by andisiambuku** [comment]
Hello @Kilgamesh I am Andisi, and I want to assist you in editing the README for the repository. I can undertake the tasks mentioned and I can also include a user guide and also pictures of the application to demonstrate to the user.

**Comment by accidental-feature** [comment]
@andisiambuku Sorry for the delay in my response. I am actually going to make a few improvements to the project within the next couple of weeks. And once I do I welcome any assistance in the documentation of the project.

If you are still interested!

**Comment by andisiambuku** [comment]
Yes I am still interested, you will let me know when you are ready
